lib/google_maps_service/apis/roads.rb

```diff
@@ -63,8 +63,9 @@ def snap_to_roads(path, interpolate: false)
     #         by the snap_to_roads function. You can pass up to 100 Place IDs.
     #
     # @return [Array] Array of speed limits.
-    def speed_limits(place_ids)
+    def speed_limits(place_ids, speed_unit='KPH')
```

Comment:
I prefer to use `units`, instead of `speed_unit`. Here I want to keep the params similar to the Google Maps request params name.

And also, please use `:`, instead of `=`, to keep consistent with other methods.